aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,58 @@
+            },
+            "AbortIncompleteMultipartUpload": {
+                "DaysAfterInitiation": 1
+            }
+        }
+    ]
+}
+
+try:
+    s3.put_bucket_lifecycle_configuration(
+        Bucket=bucket_name,
+        LifecycleConfiguration=lifecycle_config
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)